scripts/run_acceptance_tests.py

```diff
@@ -99,6 +99,21 @@ def compile_test_ts_files() -> None:
         os.path.join(build_dir_path, 'images'))
 
 
+def print_test_output(output_lines: List[bytes]) -> None:
```

Comment:
I do think this is all very complex. You can just print all the lines.

In the event of a failure, I don't think this will print the call stack as well. Which we would want to see as it tells you where and why the test failed.